tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -73,6 +73,9 @@ pub mod pallet {
 		/// maximum delay in block numbers between creating an identity and verifying an identity
 		#[pallet::constant]
 		type MaxVerificationDelay: Get<ParentchainBlockNumber>;
+		/// maximum number of identities an account can have, if you change this value to lower some accounts may exceed this limit
+		#[pallet::constant]
+		type AccountIdentitiesLimit: Get<u32>;
```

Comment:
Right now account can have in total (limit + 1) identities because prime identity is not counted. I think we should count it too, then user will be able to create (limit -1) identities because prime is created automaticaly while shielding key is set.

Reply:
Then we will be able to simplify id_graph_stats logic and use counters there.

Reply:
One question, if we limit the quantity, then the number of identities we retrieve from idgraph will be limited to this quantity, and will no longer be the latest 20 identities like before, is that correct?

Reply:
The scope of changes in this PR is mainly focused on preventing account from creating more idenitites than given limit.
In my opinion getter call used to retrieve latest 20 identities shouldn't be affected by these changes. How are they requested ?
In case of lowering the limit, you will be still able to use your 20 identities. At the worst scenario some accounts may have more identities than allowed, maybe we should consider different naming if it's confusing.